templates/selenium3.141.x-python37/src/__ALGO___test.py

```diff
@@ -4,5 +4,3 @@ def test_algorithm():
     input = {}
```

Comment:
This seems like a useless test. I think it would be cooler if you passed the in the url, find_by_element and input as parameters and did the checks that you have in the apply method here instead. And if you are returning the title/body in apply, that will be easy